libr/bin/p/bin_dyldcache.c

```diff
@@ -466,7 +543,7 @@ static RDyldRebaseInfo *get_rebase_info(RBinFile *bf, RDyldCache *cache) {
 		rebase_info->start_of_data = start_of_data;
 		rebase_info->one_page_buf = one_page_buf;
 		rebase_info->page_size = 4096;
-		rebase_info->slide = estimate_slide (bf, cache, 0xffffffffffffffffL);
+		rebase_info->slide = estimate_slide (bf, cache, 0xffffffffffffffffULL);
```

Comment:
Can't you use defines like UT64_MAX like defined in https://github.com/radare/radare2/blob/master/libr/include/r_types_base.h ?

Reply:
you're right: https://github.com/radare/radare2/pull/12720

Reply:
great thanks 👍